media-gfx/qview/qview-3.0.recipe

```diff
@@ -0,0 +1,78 @@
+SUMMARY="Minimalist image viewer"
+DESCRIPTION="qView is an image viewer designed with minimalism and usability \
+in mind.
+
+qView was designed from the very start to be as visually minimal and space \
+efficient as possible. No cluttered interface, just your image and a titlebar.
+
+qView opens practically instantly, and switching images is just as quick, all \
+while having low memory and CPU usage.
+
+With features such as animated GIF controls, file history, rotation/mirroring, \
+and multithreaded preloading, minimalism doesn't have to be a compromise."
+HOMEPAGE="https://interversehq.com/qview/"
+COPYRIGHT="2018-2020 jurplel and qView contributors"
+LICENSE="GNU GPL v3"
+REVISION="1"
+SOURCE_URI="https://github.com/jurplel/qView/archive/$portVersion.tar.gz"
+CHECKSUM_SHA256="5bc2a2eb3aa6329fe6f207a72497f115625d84a0e76663872fd2e401ea7c5ae1"
```

Comment:
The checksum may be from an older qview version? It's now:
`0c4814171c787aa361e7c1da81babad86c6aa72a42247eab537b307a7b36a24d`